Source/WTF/wtf/FixedVector.h

```diff
@@ -154,8 +154,12 @@ class FixedVector {
     T& operator[](size_t i) LIFETIME_BOUND { return m_storage->at(i); }
     const T& operator[](size_t i) const LIFETIME_BOUND { return m_storage->at(i); }
 
+    T* data() LIFETIME_BOUND { return m_storage->data(); }
```

Comment:
But we removed those to encourage people to use `span()` iirc. And you could always use `span().data()` if you *really* need a raw pointer.